src/Peachpie.CodeAnalysis/FlowAnalysis/ConditionBranch.cs

```diff
@@ -28,9 +28,22 @@ internal static class ConditionBranchEnum
         /// <summary>
         /// Switches <see cref="ConditionBranch.ToTrue"/> and <see cref="ConditionBranch.ToFalse"/>.
         /// </summary>
-        public static ConditionBranch NegativeBranch(ConditionBranch branch)
+        public static ConditionBranch NegativeBranch(this ConditionBranch branch)
         {
             return (ConditionBranch)(-((int)branch));
         }
+
+        public static bool? ToBool(this ConditionBranch branch)
```

Comment:
ah, so it is not a conversion to a Boolean but rather its expected result.. maybe rename it to something else ? like `TargetValue` ?